controller/controller_server/csi_controller_server.py

```diff
@@ -514,8 +514,15 @@ def _get_snapshot_final_name(self, volume_parameters, name, array_mediator):
                                            config.SNAPSHOT_TYPE_NAME)
         return name
 
+    def _get_version_letter(self):
+        version = self.get_identity_config("version")
+        _, minor_version, _ = version.split(config.VERSION_DELIMITER)
```

Comment:
[`from packaging.version import Version`](https://github.com/IBM/ibm-block-csi-driver/blob/20605cc8ce2249e77f3b6d71b6545ed9928bccf1/controller/array_action/array_mediator_svc.py#L5)
```suggestion
        raw_version = self.get_identity_config("version")
        version = Version(raw_version)
        minor_version = version.minor
```

Reply:
done

Reply:
done